src/renderer/src/routes/modals/CreateWallet/CreateWalletStep1Modal.tsx

```diff
@@ -75,6 +75,7 @@ export const CreateWalletStep1Modal = () => {
             label={t('nextButtonLabel')}
             flat
             onClick={() => modalNavigate('create-wallet-step-2', { state: { words } })}
+            autoFocus
```

Comment:
Autofocus here will cause the screen reader user to move to the next step without reading the copy and print buttons, and mnemonic text. I would remove it.